spec/controllers/medicaid/income_expenses_job_number_controller_spec.rb

```diff
@@ -1,33 +0,0 @@
-require "rails_helper"
-
-RSpec.describe Medicaid::IncomeJobNumberController do
```

Comment:
because of weird naming, this was a duplicate spec file (we also have specs for this controller in the correct location)